doc(network): remove references to outdated NetworkPolicy examples (#955)
longhorn/website: content/docs/1.5.0/references/networking.md

This page documents the networking communication between components in the Longhorn system. Using this information, users can write Kubernetes NetworkPolicy
to control the inbound/outbound traffic to/from Longhorn components. This helps to reduce the damage when a malicious pod breaks into the in-cluster network.

> The helm chart will install NetworkPolicy objects when the [networkPolicies.enabled value](https://github.com/longhorn/longhorn/blob/master/chart/values.yaml) is set to `true`.
> The manifests of these objects can be viewed in the [git repository](https://github.com/longhorn/longhorn/tree/master/chart/templates/network-policies).

Reviewer: ```suggestion
The helm chart installs NetworkPolicy objects when the [networkPolicies.enabled value](https://github.com/longhorn/longhorn/blob/master/chart/values.yaml) is set to `true`.
Manifests for these objects can be viewed in the [git repository](https://github.com/longhorn/longhorn/tree/master/chart/templates/network-policies).
```
